test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,94 @@
+require "test_helper"
+
+describe VotesController do
+  describe 'create' do
+    it 'can add a new vote' do
+      work = works(:test_album)
+      user = User.new(name: 'tester')
+
+      perform_login(user)
+
+      expect {
+        post work_votes_path(work)
+      }.must_differ 'work.votes', 1
```

Comment:
`must_differ` requires a number, so you need the count of the votes. 

```suggestion
      }.must_differ 'work.votes.count', 1
```